support/ruby/collins-shell/lib/collins_shell/monkeypatch.rb

```diff
@@ -15,7 +15,7 @@ def to_human_size
   end
 end
 
-class Fixnum
+class Integer
```

Comment:
IIRC in ruby 2.4 they reorganized the whole class hierarchy  and changed how Integer works. Does this still work with ruby <2.4 and ruby >=2.4?

Reply:
I tested with 1.9.3 and 2.4.1, both do the right thing – on 2.4 this is just _the_ class, on 1.9.3 we implement this method in the parent class so it's available for both Fixnum and Bignum.